homepage/styles.css

```diff
@@ -0,0 +1,142 @@
+html {
+    background-color: grey;
+}
+body {
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+.header {
+    width: 100%;
+    height: 500px;
+    color: white;
+    background-image: url("background.jpg");
+    background-repeat: no repeat;
+    background-size: cover;
+    border: 1px solid green;  
+} 
+.nav {
+    height: 50px;
+    width: 100%;
+    margin-bottom: 5px;
+    border-bottom: 0.5px solid lightgrey;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    font-family: 'Times New Roman', Times, serif;
+    font-size: 14px; 
+    }
+
+ul {
+  list-style-type: none;      
+  
+}
+li {
+    margin: 0 5px;
+    padding-right: 10px;
+    
+}
+a{
+    color: white;
+    text-decoration-line: none;    
+}
+a:hover {
+    border-bottom:0.5px solid red;
+}
+img {
+    height: 20px;
+    width: 20px;
+    }
+.nav-left {
+    width: 500px;   
+    display: inline-flex;
+    }   
+.nav-center {
+    width:250px;    
+    display: inline-flex;
+    font-size: 20px;
+}
+.nav-right {
+    width:350px;
+    display: inline-flex;    
+}
+.icons {
+    float: right;    
+    display: inline-flex;    
+}
+.caption {
+    margin: 100px auto;
```

Comment:
This add `100px` bottom margin as well. What will happens if I want to add few more lines of content to caption?